_posts/2020-08-13-convertextract-app.md

```diff
@@ -13,39 +13,39 @@ hidden: false
 # TL;DR
 
 Have you ever wanted to *NOT* spend hours tediously checking that **k + '** is written as **k̓** and not **k'**?
-If you said **YES!**, *Convertextract* is the app for you. With minimal technical knowledge, you can now systemically make your [ELAN](https://archive.mpi.nl/tla/elan) annotations consistent.
+If you said **YES!**, *Convertextract* is the app for you. With minimal technical knowledge, you can now systemically make your https://[ELAN](https://archive.mpi.nl/tla/elan) annotations consistent.
 
 # What you need to know to understand this post
 
-I assume that you know some background about the [g2p library](https://github.com/roedoejet/g2p).
-For the purposes of using these tools, a library is a collection of code and documentation, but if you would like to dig deeper you can check out this [Wikipedia article](https://en.wikipedia.org/wiki/Library_(computing)) 
+I assume that you know some background about the https://[g2p library](https://github.com/roedoejet/g2p).
+For the purposes of using these tools, a library is a collection of code and documentation, but if you would like to dig deeper you can check out this https://[Wikipedia article](https://en.wikipedia.org/wiki/Library_(computing)) 
```

Comment:
Hey @finguist , this isn't how you [add links in markdown](https://guides.github.com/features/mastering-markdown/#example-text:~:text=%5Blink%20to%20Google!%5D(http%3A%2F%2Fgoogle.com)). Adding https:// in the text outside the link will just show https:// but not actually do anything. This link is fine for example: `[g2p library](https://github.com/roedoejet/g2p)` and this link is not `https://[g2p library](https://github.com/roedoejet/g2p)`. This link is missing the protocol (http or https, best to [use https because it is secure](https://www.cloudflare.com/learning/ssl/why-is-http-not-secure/#:~:text=HTTPS%20is%20HTTP%20with%20encryption,uses%20HTTPS%20has%20https%3A%2F%2F.)): 

`[Aidan Pine](aidanpine.ca)`

If it's an email link like `[saratlilinukw@gmail.com](mailto:saratlilinukw@gmail.com)` it also doesn't work to add https protocol to an email, so you're fine just staying with `mailto:`